src/test/java/edu/unc/lib/boxc/migration/cdm/CdmIndexCommandIT.java

```diff
@@ -121,6 +121,26 @@ public void indexWithWarningsTest() throws Exception {
         assertOutputContains("CPD file referenced by object 604 in desc.all was not found");
     }
 
+    @Test
+    public void indexFromCsvTest() throws Exception {
+        initProject();
+        Files.createDirectories(project.getExportPath());
+
+        Files.copy(Paths.get("src/test/resources/files/cdm_export_fields.csv"), project.getFieldsPath());
```

Comment:
The `cdm_fields.csv` should not exist before calling the indexing command, it should be getting created as a side effect of the indexing.

We should assert that `cdm_fields.csv` exists after running the command though.